tensornetwork/network.py

```diff
@@ -932,12 +938,115 @@ def split_node(
     self.nodes_set.remove(node)
     return left_node, right_node, trun_vals
 
+
+  def split_node_qr(
```

Comment:
Can you add some tests that call this method and `split_node_rq` in the `tensornetwork_test`? Specifically, try adding a node, splitting it, and then recombining it and make sure its numerically close to the original tensor.